components/ui/ModalReport.vue

```diff
@@ -33,22 +33,10 @@
         <span class="label__title">Additional information</span>
```

Comment:
With the introduction of the new report page, this component can be deleted, no? Or are there still certain things using it?